shared/getDeviceOnDash/raspberrypi.md

```diff
@@ -1,10 +1,8 @@
-## Setting Up Your Device
-
-Insert the SD card into the {{ $device.name }} and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your Pi by inserting the micro USB cable.
+Insert the SD card into your {{ $device.name }} and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your Pi by inserting the micro USB cable.
```

Comment:
The `, if necessary,` here isn't in the raspberrypi2 guide? (or is this because the raspberrypi1 includes the PiZeroW, which has built-in Wi-fi?)